.github/labeler-review.yml

```diff
@@ -0,0 +1,5 @@
+Ready to be reviewed:
+- '**/*'
+
+Waiting to be tested:
+- '**/*'
```

Comment:
not sure about this, the PR might not be ready when it's opened

Reply:
Check labeler.yml or whatever the name is.

If it is in draft it wont apply

Reply:
runs-on: ubuntu-20.04
    if: (github.event.pull_request.draft == false) || (github.event.action.synchronize == false) || (github.event.action.unlabled == false) && (github.event.action.ready_for_review == true)
    steps:

Reply:
ok, is there a way to also filter for PRs that have passed the build?

Reply:
I can check in the eeek

Reply:
I think not.
But that part will only trigger if
Not a Draft PR
Not being rebased/synchronized with Master (Using the button below a PR)
Not if unlabled
and if it is ready for review (Not in draft, don't know why they have two different hooks for this)
@FrancescoBorzi